BIDebugEngine/BIDebugEngine/Debugger.cpp

```diff
@@ -484,8 +474,9 @@ void Debugger::checkForBreakpoint(DebuggerInstructionInfo& instructionInfo) {
     if (doBreeak && instructionInfo.instruction->sdp.sourcefile.find("dedmen\\") != std::string::npos) __debugbreak();
     auto space = instructionInfo.instruction->sdp.sourcefile.find("[");
     auto properPath = (space != std::string::npos) ? instructionInfo.instruction->sdp.sourcefile.substr(0, space-1) : instructionInfo.instruction->sdp.sourcefile;
-    auto &found = breakPoints.get(properPath);
-    if (breakPoints.is_null(found) || found.empty()) return;
+    auto& found = breakPoints.get(static_cast<std::string>(properPath).c_str());
```

Comment:
again, copying into string every time is not an option.

Reply:
I didn't mean to change this in the PR I thought I changed it back.